learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -165,7 +165,7 @@ Nodes and Terminology
 Before continuing, it must be noted that the *Node* termonology needs to be used with care. 
```

Comment:
Would you mind fixing some more? :P
termonology --> terminology
refering --> referring

Reply:
there you go #462